### PR TITLE
Documentation fixes

### PR DESCRIPTION
**Pull request by waynetee**
Made various fixes to DG and PPP
preview at https://waynetee.github.io/tp/DeveloperGuide.html

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -557,131 +565,121 @@ Use case ends.
...
+3. `PropertyWhiz` exports displayed properties/buyers to the CSV file.
 
 Use case ends.
 
 **Extensions**
 
-* 2a. `User` cancels while choosing a csv file.
-    * 2a1. `PropertyWhiz` shows an error message.
+* 1a. `User` cancels while choosing a CSV file.
+    * 1a1. `PropertyWhiz` shows an error message.
```
Comment (trxe) [inline docs/DeveloperGuide.md:630]: The message "Import properties was cancelled." isn't really an error messge, maybe "`PropertyWhiz` reports that import is cancelled"

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -557,131 +565,121 @@ Use case ends.
...
-
 ### Non-Functional Requirements
 
 1. Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2. Should be able to hold up to 500 properties and buyers without a noticeable sluggishness in performance for typical usage.
+2. Should be able to hold up to 200 properties and buyers without a noticeable sluggishness in performance for typical usage.
 3. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
 4. Should be able to function without internet access.
 5. Should be able to restart without loss of data.
 6. Should be packaged within a single jar, without requiring an installer.
```
Comment (trxe) [inline docs/DeveloperGuide.md:675]: "single JAR file"

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -699,7 +697,7 @@ testers are expected to do more *exploratory* testing.
 1. Initial launch
 
    1. Download the jar file and copy into an empty folder
-   2. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
+   2. Double-click the jar file Expected: Shows the GUI with a set of sample properties and buyers. The window size may not be optimum.
```
Comment (trxe) [inline docs/DeveloperGuide.md:700]: should uppercase all "jar" i.e. JAR to be consistent with "csv"

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -724,31 +722,31 @@ testers are expected to do more *exploratory* testing.
...
 
 ### Exporting and importing data
 
-1. Testing properties export/import csv
+1. Testing properties export/import CSV
    1. Remove `propertywhiz.json` file in the /data folder.
    2. Run `PropertyWhiz` to obtain an initialized list of properties and buyers.
-   3. Enter Command: `export properties`, select a csv file at a convenient location..<br>
-      Expected: csv file is created, containing a header and 1 property per row.
+   3. Enter Command: `export properties`, select a CSV file at a convenient location.<br>
```
Comment (trxe) [inline docs/DeveloperGuide.md:744]: `export properties` -> `export property` to be consistent with UG

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -724,31 +722,31 @@ testers are expected to do more *exploratory* testing.
...
-      Expected: Similar to previous.
+   4. Other incorrect delete commands to try: `delete`, `delete property x` (where x is larger than the list size)<br>
+      Expected: Similar to previous test cases.
 
 4. Sort properties that are in the displayed list
    1. Test case: `sort property name desc`<br>
       Expected: Sorted all properties in the displayed properties list by name in descending order.
    2. Test case: `sort property name`<br>
-      Expected: There is no sort direction given. Properties are not sorted.
+      Expected: As there is no sort direction given, properties are not sorted.
```
Comment (sunjc826) [inline docs/DeveloperGuide.md:737]: This should be "Error message shown"?

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -724,31 +722,31 @@ testers are expected to do more *exploratory* testing.
...
+   3. Enter Command: `export properties`, select a CSV file at a convenient location.<br>
+      Expected: CSV file is created, containing a header and 1 property per row.
    4. Enter Command: `clear` to clear existing properties and buyers.
-   5. Enter Command: `import properties` from the csv file.<br>
+   5. Enter Command: `import properties` from the CSV file.<br>
       Expected: Properties are successfully imported.
-   6. Enter Command: `import properties` from the csv file.<br>
+   6. Enter Command: `import properties` from the CSV file.<br>
       Expected: Import properties failed due to duplicates.
 2. Repeat the commands above to test for buyers.
```
Comment (trxe) [inline docs/DeveloperGuide.md:751]: For the section **Testing multiple commands quickly**, add one more `back` command at line 808

**Review by sunjc826 — COMMENTED** [review]
The displayed lists are a key part of the app's design, even if they are UI details.
What does @guangxuan think?
I am not too sure about their removal from the use cases.

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -832,11 +830,11 @@ If AB3 required an implementation effort of 10, `PropertyWhiz`’s implementatio
...
-* Import and export csv file commands
+* Import and export CSV file commands
   * A new workflow was required for the UI to handle files. We must first parse the command text to check if it requires a file, request for a file if needed, then consider executing the command. 
   * Solution: we added a new type of command- `CommandWithFile` to represent commands that require a file. We then added `CommandPreAction` to indicate that a FileChooserDialog is required for the user to select a file. 
-  * `CsvManager` was added as a static, purely functional class. The parsing of csv file was tricky and involved in multiple exceptions. Careful analysis of cases was required to display an appropriate error message.
-  * Here, [opencsv](http://opencsv.sourceforge.net/) played an important role in dealing with the huge number of possible errors while parsing csv files.
+  * `CsvManager` was added as a static, purely functional class. The parsing of CSV file was tricky and involved in multiple exceptions. Careful analysis of cases was required to display an appropriate error message.
```
Comment (trxe) [inline docs/DeveloperGuide.md:836]: ... CSV file**s** was tricky and was involved in multiple...

**Review by trxe — APPROVED** [review]
LGTM after you fix the nitpicks, thanks so much!

**Comment by trxe** [comment]
One more thing, there's still a TODO under **Model Component**, I think we should just remove it

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -422,13 +422,13 @@ Here are several extensions that can be implemented in the future:
 
 **Target user profile**:
 
-* has a need to manage a significant number of properties and buyers
-* prefer desktop apps over other types
+* property agents who need to manage a significant number of properties and buyers
+* prefer desktop apps
 * can type fast
 * prefers typing to mouse interactions
 * is reasonably comfortable using CLI apps
```
Comment (lizchow) [inline docs/DeveloperGuide.md:429]: Can we capitalise the first letter of the sentence?

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -557,131 +565,121 @@ Use case ends.
...
-* 3b. The csv file is incorrectly formatted.
-    * 3b1. `PropertyWhiz` shows an error message.
+* 2b. The CSV file is incorrectly formatted.
+    * 2b1. `PropertyWhiz` shows an error message.
 
     Use case ends.
 
-* 3c. Importing creates duplicate properties within `PropertyWhiz`.
-    * 3c1. `PropertyWhiz` shows an error message.
+* 2c. Importing creates duplicates within `PropertyWhiz`.
```
Comment (lizchow) [inline docs/DeveloperGuide.md:610]:
```suggestion
* 2c. Importing causes duplicates within `PropertyWhiz`.
```

File: docs/DeveloperGuide.md [hunk 10]
```diff
@@ -557,131 +565,121 @@ Use case ends.
...
-
-**Use case: UC09 - Stat Properties**
+**Use case: UC09 - Charting**
 
 **MSS**
 
-1. `User` requests to draw a histogram of property prices.
-2. `PropertyWhiz` draws and shows in a new window, a histogram of property prices within the displayed property list. 
+1. `User` requests to for a chart of prices of properties/buyers.
+2. `PropertyWhiz` draws and displays chart showing prices of properties/buyers.
```
Comment (lizchow) [inline docs/DeveloperGuide.md:664]:
```suggestion
1. `User` requests for a chart of prices of properties/buyers.
2. `PropertyWhiz` draws and displays chart showing prices of properties/buyers in a new window.
```

File: docs/DeveloperGuide.md [hunk 11]
```diff
@@ -724,31 +722,31 @@ testers are expected to do more *exploratory* testing.
...
+      Expected: No property is deleted. Error message shown.
 
-   4. Other incorrect delete commands to try: `delete`, `delete property x`, `...` (where x is larger than the list size)<br>
-      Expected: Similar to previous.
+   4. Other incorrect delete commands to try: `delete`, `delete property x` (where x is larger than the list size)<br>
+      Expected: Similar to previous test cases.
 
 4. Sort properties that are in the displayed list
    1. Test case: `sort property name desc`<br>
       Expected: Sorted all properties in the displayed properties list by name in descending order.
```
Comment (lizchow) [inline docs/DeveloperGuide.md:735]:
```suggestion
      Expected: Sorted all properties in the displayed property list by name in descending order.
```

File: docs/DeveloperGuide.md (hunk 10 above)
Comment (waynetee) [reply docs/DeveloperGuide.md:664]: Ui details though

File: docs/DeveloperGuide.md (hunk 9 above)
Comment (waynetee) [reply docs/DeveloperGuide.md:610]: I think creates sounds better to me, what do the others think? @AY2122S1-CS2103T-W11-4/developers

**Pull request merged**